lib/transport/selenium-webdriver/options.js

```diff
@@ -102,6 +102,14 @@ module.exports = class SeleniumCapabilities {
       return this.desiredCapabilities;
     }
 
+    // Backward compatibility for 'chromeOptions'
+    if (this.desiredCapabilities.chromeOptions) {
+      this.desiredCapabilities['goog:chromeOptions'] = Object.assign(
+        this.desiredCapabilities['goog:chromeOptions'] || {},
+        this.desiredCapabilities.chromeOptions
+      );
+    }
```

Comment:
I think it should handle the case where a property is defined in both the object but it should consider the one defined in `goog:chromeOptions`

```suggestion
    // Backward compatibility for 'chromeOptions'
    this.desiredCapabilities['goog:chromeOptions'] = Object.assign(this.desiredCapabilities.chromeOptions || {}, this.desiredCapabilities['goog:chromeOptions'] )
```

Reply:
Makes sense.

Reply:
Done.